deploy/basic/README.md

```diff
@@ -1,8 +1,12 @@
 ![MuShop Logo](../../images/logo.png)
 
-This is a Terraform configuration that deploys the MuShop Basic demo on [Oracle Cloud Infrastructure (OCI)][oci].
+This is a Terraform configuration that deploys the MuShop application demo on [Oracle Cloud Infrastructure (OCI)][oci].
```

Comment:
remove `demo`

Reply:
And keep the basic notation, as will be different application soon.

Reply:
Didnt we agree to keep the "application demo" like Hispter shop ?

Reply:
MuShop basic sample application